06. Errors&Exceptions/01. OwnLogDecorator.py

```diff
@@ -0,0 +1,24 @@
+# 1. Write your own decorator, the task of which should be to print the name of the function and the time when it was
+# called. The decorator should work the same for different functions.
+import datetime
+
+
+def get_log(func):
+    def wraps(*args, **kwargs):
+        func(*args, **kwargs)
```

Comment:
It is better to use `return` statement here to be able to decorate functions that return some values